hockeysdk/src/main/java/net/hockeyapp/android/objects/CrashDetails.java

```diff
@@ -56,6 +56,10 @@
 
     public CrashDetails(String crashIdentifier) {
         this.crashIdentifier = crashIdentifier;
+        appStartDate = new Date();
```

Comment:
No need for a new PR, just update this one to, so there's only

```
isXamarinException = false;
throwableStackTrace = "";
```

Otherwise this looks good to me.